network/src/main/java/bisq/network/p2p/services/peergroup/PeerGroup.java

```diff
@@ -85,14 +87,29 @@ public PeerGroup(Node node, Config config, List<Address> seedNodeAddresses, BanL
 
     public void addReportedPeers(Set<Peer> peers) {
         reportedPeers.addAll(peers);
+        addPersistedPeers(peers); // New reported peers are also persisted
```

Comment:
I guess we need a bit of refactoring here so that the caller is using the service for updating the model and the service is calling persist() after the PeerGroup model has been changed.

Reply:
Alternatively we could redefine PeerGroup as sub-servive and PeerGroupService is more the coordination/managing parent service and a new model will be created for the PeerGroup data. I will have a look to the code to get a better idea, whats the best option here.